gvmtools/helper.py

```diff
@@ -113,6 +116,33 @@ def error_and_exit(msg):
     sys.exit(1)
 
 
+def generate_random_ips(count: int):
+    """Generate count random IPv4s"""
+    exclude_127 = [i for i in range(1, 256)]
+    exclude_127.remove(127)
+    return [
+        '{}.{}.{}.{}'.format(
+            choice(exclude_127),
+            randrange(0, 256),
+            randrange(0, 256),
+            randrange(1, 255),
+        )
+        for i in range(count)
+    ]
+
+
+def generate_random_id(
```

Comment:
I would prefer `generate_id` over `generate_random_id` since when I want to generate an id I usually don't mind if it's randomly chosen or generated by some sophisticated rule.